native_client/python/client.py

```diff
@@ -37,20 +39,15 @@
 N_CONTEXT = 9
 
 def convert_samplerate(audio_path):
-    sox_cmd = 'sox {} --type raw --bits 16 --channels 1 --rate 16000 - '.format(audio_path)
+    sox_cmd = 'sox {} --type raw --bits 16 --channels 1 --rate 16000 - '.format(shlex.quote(audio_path))
     try:
-        p = subprocess.Popen(sox_cmd.split(),
-                             stderr=subprocess.PIPE, stdout=subprocess.PIPE)
-        output, err = p.communicate()
-
-        if p.returncode:
-            raise RuntimeError('SoX returned non-zero status: {}'.format(err))
-
+        output = subprocess.check_output(shlex.split(sox_cmd), stderr=subprocess.PIPE)
```

Comment:
I think there was some trickyness on the way bytes / strings gets treated, and this is why we were doing `stdout=subprocess.PIPE` and `communicate()` instead of just `check_output`